src/networking.c

```diff
@@ -3280,13 +3281,17 @@ int processInputBuffer(client *c) {
 
 /* This function can be called from the main-thread or from the IO-thread.
  * The function allocates query-buf for the client if required and reads to it from the network.
- * It will set c->nread to the bytes read from the network. */
-void readToQueryBuf(client *c) {
+ * It will set c->nread to the bytes read from the network.
+ * Returns non-zero if the buffer was filled (more data may be available).
+ */
+
```

Comment:
Does it return bool or int? Let's stick to one of them in wording and types.

```suggestion
 * Returns true if the buffer was filled (more data may be available). */
```